jumpcloud-ADMU/Powershell/Tests/manualTests/selectionForm.feature

```diff
@@ -0,0 +1,217 @@
+        When a domain profile is selected
+        And the "Force Reboot" checkbox is selected
+        And a JumpCloud username is specified
+        And a password is specified
+        Then the "Migrate Profile" button should become active and no longer be greyed-out
+
+    Scenario: Allows "Migrate Profile" Button with LeaveDomain, domain profile, username and password
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "Leave Domain" checkbox is selected
+        And a JumpCloud username is specified
+        And a password is specified
+        Then the "Migrate Profile" button should become active and no longer be greyed-out
+
+Feature: Selection form disallows user migration for invalid JumpCloud Usernames
+
+    Scenario: Disallows "Migrate Profile" Button for JumpCloud Usernames that are null
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And a JumpCloud username text "" is specified in the text box
+        And a password is specified
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+    Scenario: Disallows "Migrate Profile" Button for JumpCloud Usernames with spaces in the names
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And a JumpCloud username text "some user" is specified in the text box
+        And a password is specified
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+    Scenario: Disallows "Migrate Profile" Button for JumpCloud Usernames longer than 20 characters
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And a JumpCloud username text "aVeryLongUserNameIsInvalid" is specified in the text box
+        And a password is specified
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+    Scenario: Disallows "Migrate Profile" Button for JumpCloud Usernames that already exist on the device
+        Given a new local user has been created on the device with username "newUser" (Use `New-LocalUser` or `lusrmgr.msc` to create a new user)
+        And The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And a JumpCloud username text "newUser" is specified in the text box
+        And a password is specified
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+    Scenario: Disallows "Migrate Profile" Button for JumpCloud Usernames that match the hostname
```

Comment:
Username textbox should have the error icon if the hostname is inputted 
<img width="367" alt="image" src="https://github.com/user-attachments/assets/6dc77229-7bc8-4fa7-8008-1bebe6f7e15d" />

Reply:
should be resolve in 32110517dbf7ae30d0134626eb6232f2a4c34024